video/out/w32_common.c

```diff
@@ -963,6 +963,16 @@ static bool is_visible(HWND window)
     return GetWindowLongPtrW(window, GWL_STYLE) & WS_VISIBLE;
 }
 
+//Set the mpv window's affinity.
+//This will affect how it's displayed on the desktop and in system-level operations like taking screenshots.
+static bool update_affinity(struct vo_w32_state *w32)
```

Comment:
Since this bool isn't ever used anywhere, why not make it void?

Reply:
I left it bool for possible future use, but I guess we can for now just void it